site/index.html

```diff
@@ -1,11 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+<head>
+  <link rel="icon" type="image/png" href="/C:\Users\Nanab\Desktop\NK_Repo_Lab1/favicon-96x96.png" sizes="96x96" />
+<link rel="icon" type="image/svg+xml" href="/C:\Users\Nanab\Desktop\NK_Repo_Lab1/favicon.svg" />
+<link rel="shortcut icon" href="/C:\Users\Nanab\Desktop\NK_Repo_Lab1/favicon.ico" />
+<link rel="apple-touch-icon" sizes="180x180" href="/C:\Users\Nanab\Desktop\NK_Repo_Lab1/apple-touch-icon.png" />
+<link rel="manifest" href="/C:\Users\Nanab\Desktop\NK_Repo_Lab1/site.webmanifest" /> 
```

Comment:
Prefer relative urls
```suggestion
<head>
  <link rel="icon" type="image/png" href="favicon-96x96.png" sizes="96x96" />
<link rel="icon" type="image/svg+xml" href="favicon.svg" />
<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
<link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
<link rel="manifest" href="site.webmanifest" /> 
```